tests/basic/test_float.js

```diff
@@ -0,0 +1,21 @@
+"use strict";
+
+function assert(actual, expected, message) {
+    if (arguments.length == 1)
+        expected = true;
+
+    if (actual == expected)
```

Comment:
use "==="

Reply:
I think there is a BUG in the code, because during testing I found that -1.1.toString() == "-1.100000" passed, but -1.1.toString() === "-1.100000" failed.

Reply:
I'll try to find the reason in another PR

Reply:
```
qjs > let b = -1.1.toString()
undefined
qjs > typeof(b)
"number"
```